### PR TITLE
Add instructions to README

### PR DESCRIPTION
**Pull request by 0xp3th1um**
- Merkle root generation instruction
- Set up hardhat vars instructions
- Deployment instructions

### CODE REVIEW

**Review by oddaf — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -18,3 +18,27 @@ The following assumes the use of `node@>=10`.
...
 `yarn test`
+
+## Generate Merkle Tree
+
+`npx ts-node  scripts/generate-merkle-root.ts -i "input-json-file"`
+
+
+## Deploy Merkle distributor
+
+First, set-up the configuration variables using Harhad:
```
Comment (amusingaxl) [inline README.md]:
```suggestion
First, set-up the configuration variables using Hardhat:
```

File: README.md [hunk 2]
```diff
@@ -18,3 +18,27 @@ The following assumes the use of `node@>=10`.
...
+
+`npx hardhat vars set XXX_PRIVATE_KEY`
+
+`npx hardhat vars set XXX_RPC_URL`
+
+`npx hardhat vars set TENDERLY_CHAIN_ID`
```
Comment (amusingaxl) [inline README.md]:
```suggestion
`npx hardhat vars set XXX_CHAIN_ID`
```

File: README.md (hunk 1 above)
Comment (0xp3th1um) [reply README.md]: good catch

File: README.md (hunk 2 above)
Comment (0xp3th1um) [reply README.md]: This is intentional...because for mainnet it is not used (it is always 1) but we can add it if you prefer to be explicit.

Comment (amusingaxl) [reply README.md]: Maybe then add something like: "If using Tenderly Virtual Testnets, provide TENDERLY_CHAIN_ID".

**Review by oddaf — APPROVED** [review]

**Review by amusingaxl — APPROVED** [review]

**Pull request merged**